sdk/storage/storage-internal-avro/src/utils/utils.common.ts

```diff
@@ -0,0 +1,26 @@
+export interface Metadata {
+  /**
+   * A name-value pair.
+   */
+  [propertyName: string]: string;
+}
+
+export interface Dictionary<T> {
```

Comment:
Could be `type Dictionary<T> = Record<string, T>`, although not sure the alias offers much (same as above).

Reply:
Changed to use `Record` and removed the `Dictionary` interface.